service/resource/resourcegroup/error.go

```diff
@@ -18,9 +18,9 @@ func IsWrongTypeError(err error) bool {
 	return microerror.Cause(err) == wrongTypeError
 }
 
-var deleteTimeoutError = microerror.New("delete timeout")
+var timeoutError = microerror.New("timeout")
 
-// IsDeleteTimeoutError asserts deleteTimeoutError.
-func IsDeleteTimeoutError(err error) bool {
-	return microerror.Cause(err) == deleteTimeoutError
+// IsTimeoutError asserts deleteTimeoutError.
+func IsTimeoutError(err error) bool {
+	return microerror.Cause(err) == timeoutError
 }
```

Comment:
Ninja fix: this did not have sense when the error is nicely annotated. Also we go towards generic direction.